orangecontrib/text/datasets/url.tab

```diff
@@ -0,0 +1,19 @@
+topic	url_case_study	url

```

Comment:
If there is no relation between `url_case_study` and `url`, then maybe it would be better to have each in it's own row?